docs/DeveloperGuide.md

```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Customise Order Feature
+
+#### Implementation
+
+This feature has 2 crucial methods for this feature are in the `CustomiseCommand` and `PersonCard` classes.
+
+* `CustomiseCommand#changeAttributeOrder(Model model)` - saves the new attribute order in the `preferences.json`
+* `PersonCard#setAttributes()` - reads the order from the `preferences.json` file and builds the person card in the order specified.
+
+_{diagrams and more in depth explanation to be added}_
+
+#### Design considerations:
```

Comment:
`Design consideration`, removing the `s:` to make the TOC look nicer

Reply:
Ok, I have removed the `s`.